.github/workflows/.test.yml

```diff
@@ -0,0 +1,427 @@
+# reusable workflow
+name: .test
+
+# TODO: hide reusable workflow from the UI. Tracked in https://github.com/community/community/discussions/12025
+
+on:
+  workflow_call:
+    inputs:
+      use_snapshotter:
+        required: true
+        type: boolean
+      storage_driver:
+        required: true
+        type: string
```

Comment:
Do you see general value in making this an input, rather than simply choosing `overlay2` or `overlayfs` for `DOCKER_GRAPHDRIVER` based on `use_snapshotter`?

Reply:
No real value no, you're right, I'll remove the storage driver input